google/cloud/sqlalchemy_spanner/sqlalchemy_spanner.py

```diff
@@ -165,9 +167,13 @@ def visit_DECIMAL(self, type_, **kw):
         return "NUMERIC"
 
     def visit_VARCHAR(self, type_, **kw):
+        if type_.length is None:
+            type_.length = 40
```

Comment:
Added default value of String column, when value is `None`, spanner throws an error.

Reply:
Having a default length of 40 seems rather arbitrary. Where did you get this number?

Reply:
All other tests having that number so for temporary put the 40  and mentioned for the suggestion.

Reply:
Please use `MAX' as the default length